src/test/java/org/painandsuffer/items/weapons/DaggerTest.java

```diff
@@ -0,0 +1,26 @@
+package org.painandsuffer.items.weapons;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class DaggerTest {
+
+    @Test
+    public void whenDaggerCreatedWithDefaultDamageAndDurability_thenDaggerCreatedWithDamage10AndDurability100(){
+        Dagger dagger = new Dagger();
+        int daggerDamage = 10;
+        int daggerDurability = 100;
+        assertNotNull(dagger);
+        assertEquals(dagger.getDamage(),daggerDamage);
+        assertEquals(dagger.getDurability(),daggerDurability);
+    }
+
+    @Test
+    public void whenDaggerUseDescription_thenDescriptionAppears(){
```

Comment:
Applicable to all Weapon test classes.
Test verify that the description method returns a non-null string. However, this test don't assert anything about the content of the description, which means they could be improved to ensure the description is meaningful and correct.